crawler.py

```diff
@@ -559,7 +587,7 @@ def merge_saved_data(self):
 
         return {'version': self.version, 'snitch_map': snitch_map}
 
-    def crawl(self):
+    def crawl(self): # noqa: C901
```

Comment:
No need to ignore the complexity warning if we avoid the if/else for logging as suggested above.